.omeroci/app-config

```diff
@@ -0,0 +1,6 @@
+set -e
+set -u
+OMERO_VENV=${OMERO_VENT:-/opt/omero/web/venv}
+OMERO_DIST=${OMERO_DIST:-/opt/omero/web/OMERO.web}
+${OMERO_VENV}/bin/python \
+    $OMERO_DIST/bin/omero config append omero.web.apps '"omero_webtest"'
```

Comment:
Either as part of https://github.com/openmicroscopy/omero-test-infra/pull/5 or as a follow-up we should have the correct `python` as well as `omero` on the `$PATH` so this boilerplate is not needed.

Reply:
@jburel have you tried collapsing these 4 lines to `omero config append omero.web.apps '"omero_webtest"'`?

Reply:
As indicated on another PR, the plan is to remove the default config to the test-infra

Reply:
Understood, can you cross-link to the reference PR or even card recording the series of step?

Reply:
https://github.com/ome/omero-fpbioimage/pull/19